packages/material/scss/spreadsheet/_layout.scss

```diff
@@ -52,4 +52,10 @@
         }
     }
 
+    // Spreadsheet window
+    .k-spreadsheet-window {
+        .k-external-dropzone {
+            margin-block-end: k-map-get( $kendo-spacing, 2 );
+        }
+    }
```

Comment:
Remove this, as this will be inherited from the Default theme. If you need to use different margin value, extract a variable for this.